ISA-XLSX.md

```diff
@@ -0,0 +1,741 @@
+| Study Assay Measurement Type Term Accession Number | String     | The accession number from the Term Source associated with the selected term.                                                                                                                                                                                                                                        |
+| Assay Measurement Type Term Source REF       | String     | The Source REF has to match one of the Term Source Name declared in the Ontology Source Reference section.                                                                                                                                                                                                          |
+| Assay Technology Type                        | String     | Term to identify the technology used to perform the measurement, e.g. DNA microarray, mass spectrometry. The term can be free text or from, for example, a controlled vocabulary or an ontology. If the latter source is used the Term Accession Number and Term Source REF fields below are required.              |
+| Assay Technology Type Term Accession Number  | String     | The accession number from the Term Source associated with the selected term.                                                                                                                                                                                                                                        |
+| Assay Technology Type Term Source REF        | String     | Identifies the controlled vocabulary or ontology that this term comes from. The Source REF has to match one of the Term Source Names declared in the Ontology Source Reference section.                                                                                                                             |
+| Assay Technology Platform                    | String     | Manufacturer and platform name, e.g. Bruker AVANCE                                                                                                                                                                                                                                                                  |
+| Assay File Name                              | String     | A field to specify the name of the Assay Table file corresponding the definition of that assay. There can be only one file per cell.                                                                                                                                                                                |
+
+**Example**
+
+For example, the `ASSAY` section of an ISA-XLSX `isa.assay.xlsx` file may look as follows:
+
+|                     |                                      |
+|---------------------|--------------------------------------|
+| ASSAY |
+| Assay File Name     | assays/Proteomics/isa.assay.xlsx     | 
```

Comment:
I'm wondering whether this should be **relative** in the study and assay files. Or basically relative in all. 

So in isa.investigation.xlsx it would be `assays/Proteomics/isa.assay.xlsx`. 
And in the top-level metadata`assays/Proteomics/isa.assay.xlsx`:`isa_assay` it should be `isa.assay.xlsx`?

Reply:
Relative to top-level root is my current modus-operandi for tooling. 

I will try to incorporate specifications about this in the next patch (1.2.1)/minor (1.3.0) release. 

I think this is something for the `arc specification` though. In ISA-XLSX it could be done in any other way, if used outside of the ARC context.